invenio_files_rest/models.py

```diff
@@ -286,7 +286,7 @@ def get_by_name(cls, name):
         """Fetch a specific location object."""
         return cls.query.filter_by(
             name=name,
-        ).one()
+        ).one_or_none()
```

Comment:
What's the reason behind this change? I mean, it will go from throwing an exception to returning None, so any code relying on an exception will break

Reply:
It's kind of two folded:

- Consistency: All the other `get` operations in the models.py return a `one_or_none`. We didn't understood why this one exactly was returning just `one`.
- Makes the CLI code simpler. We just check for `None` istead of `try/except`.

Nonetheless, I understand that this would make it a minor version. However, since the CLI has breaking changes two (`location` becomes a group not a command). It would need it anyway, no?